Xamarin.Forms.Platform.iOS/Renderers/LabelRenderer.cs

```diff
@@ -213,7 +213,7 @@ void ElementPropertyChanging(object sender, PropertyChangingEventArgs e)
 
 		void UpdateTextDecorations()
 		{
-			if (!Element.IsSet(Label.TextDecorationsProperty))
+			if (!Element.IsSet(Label.TextDecorationsProperty) || !(Control.AttributedText?.Length > 0))
```

Comment:
This renderer is used also for Mac. Therefore, the specific code for iOS must be wrapped in a directive `__MOBILE__`. Also need add realization for Mac i think.

Reply:
Oh, sorry for that, done!